crates/rust-analyzer/src/config.rs

```diff
@@ -2032,7 +2029,7 @@ impl Config {
 
     pub fn client_commands(&self) -> ClientCommandsConfig {
         let commands = self.commands();
-        let force = commands.is_none() && *self.lens_forceCustomCommands();
+        let force = commands.is_none();
```

Comment:
It's unrelated to this PR and not a big deal, but I think we call `client_commands` four times per hover action, and it does a bunch of allocations, so we might want to cache it.